rocAL/rocAL_pybind/examples/rocAL_api_python_unittest.py

```diff
@@ -52,154 +77,322 @@ def main():
     rocal_cpu = False if args.rocal_gpu else True
     device = "cpu" if rocal_cpu else "cuda"
     batch_size = args.batch_size
+    max_height = args.max_height
+    max_width = args.max_width
+    color_format = types.RGB if args.color_format else types.GRAY
+    tensor_layout = types.NHWC if args.color_format else types.NCHW
+    tensor_dtype = types.UINT8
```

Comment:
Why is the tensor_dtype hardcoded in unit test ?

Reply:
The tensor dtype is hardcoded in the c++ unit tests so we are doing the same here also